Views/albums.hbs

```diff
@@ -0,0 +1,7 @@
+{{#each
+  artists
```

Comment:
In your route, you passed the information as `albums` and you are trying to loop over each `artists` to render the information. Just need to change for `albums` and you'll get the data displayed.